### PR TITLE
Add verifySeed endpoint

### PR DESCRIPTION
**Pull request by OlivierChirouze**

### CODE REVIEW

File: paf-mvp-client-express/src/client-node.ts [hunk 1]
```diff
@@ -450,6 +463,37 @@ export class ClientNode extends Node {
...
+    try {
+      const isResponseValid = this.client.verifySeed(message);
+      if (!isResponseValid) {
+        // TODO [errors] finer error feedback
+        const error: NodeError = {
+          type: 'VERIFICATION_FAILED',
+          details: '',
+        };
+        this.logger.Error(jsonProxyEndpoints.verifySeed, error);
+        res.status(400);
```
Comment (MohamedAmineGhazouani) [inline paf-mvp-client-express/src/client-node.ts:477]: No need to set the status and body. It is the responsibility of the errorHandler. You can just call `next(error); `

File: paf-mvp-client-express/src/client-node.ts [hunk 2]
```diff
@@ -450,6 +463,37 @@ export class ClientNode extends Node {
...
+        res.json(error);
+        next(error);
+      } else {
+        res.json(); // For the moment, send empty response
+        next();
+      }
+    } catch (e) {
+      this.logger.Error(jsonProxyEndpoints.verifySeed, e);
+      // FIXME finer error return
+      const error: NodeError = {
```
Comment (MohamedAmineGhazouani) [inline paf-mvp-client-express/src/client-node.ts:487]: No need to explicitly create an unknownError object. Calling the error handler with an internal error `next(e)` will do the work.

**Review by MohamedAmineGhazouani — CHANGES_REQUESTED** [review]

File: paf-mvp-client-express/src/client-node.ts (hunk 2 above)
Comment (OlivierChirouze) [reply paf-mvp-client-express/src/client-node.ts:487]: It was previous to your commit ;-)

Comment (OlivierChirouze) [reply paf-mvp-client-express/src/client-node.ts:487]: You nedd to first merge your PR so that I can do this!

Comment (MohamedAmineGhazouani) [reply paf-mvp-client-express/src/client-node.ts:487]: Not really. ErrorHandler is defined in Node.ts and it's already good on the main branch.

Comment (OlivierChirouze) [reply paf-mvp-client-express/src/client-node.ts]: Yes, but that would have been inconsistent with the rest of the code: I prefer to merge everything in one go.
Now this is done, I let you validate, thanks!

**Review by MohamedAmineGhazouani — APPROVED** [review]

**Pull request merged**